docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,139 @@ description: Dagster can orchestrate dbt alongside other technologies.
-  dbt projects. For larger projects, we recommend using
-  <a href="/integrations/dbt/reference#loading-models-using-load_assets_from_dbt_manifest">
-    <code>load_assets_from_dbt_manifest</code> to load models from a dbt{" "}
-    <code>manifest.json</code> file
-  </a>.
-</Note>
+Now that you have a Dagster project, you can run Dagster's UI to take a look at it.
 
-Let's take a look at the arguments we've supplied:
+1. Change directories to the Dagster project directory:
 
-- `project_dir`, which is the path to the dbt project
-- `key_prefix`, which is a prefix to apply to all models in the dbt project
+   ```shell
+   cd jaffle_dagster/
+   ```
 
----
+2. To start Dagster's UI, run the following:
 
-## Step 2: Define a Dagster code location
+   ```shell
+   DAGSTER_DBT_PARSE_PROJECT_ON_LOAD=1 dagster dev
+   ```
 
-Next, you'll define the code location for your Dagster project. A [code location](/concepts/code-locations), created using the <PyObject object="Definitions" /> object, is a collection of definitions in a Dagster project, such as assets, resources, and so on.
+   Which will result in output similar to:
 
-Assets loaded from dbt require a dbt resource, which is responsible for firing off dbt CLI commands. Using the <PyObject module="dagster_dbt" object="DbtCliResource" /> resource, we can supply a dbt resource to the dbt project.
+   ```shell
+   Serving dagster-webserver on http://127.0.0.1:3000 in process 70635
+   ```
 
-Open the `__init__.py` file, located in `/tutorial_template/tutorial_dbt_dagster`, and add the following code:
+3. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the assets:
 
-```python
-import os
+   <Image
+   alt="Asset graph in Dagster's UI, containing dbt models loaded as Dagster assets"
+   src="/images/integrations/dbt/using-dbt-with-dagster/load-dbt-models/dbt-asset-graph.png"
+   width={2300}
+   height={1706}
+   />
 
-from dagster_dbt import DbtCliResource
-from tutorial_dbt_dagster import assets
-from tutorial_dbt_dagster.assets import DBT_PROJECT_PATH
+---
 
-from dagster import Definitions, load_assets_from_modules
+## Step 4: Build your dbt models in Dagster
 
-resources = {"dbt": DbtCliResource(project_dir=DBT_PROJECT_PATH)}
+You can do more than view your dbt models in Dagster – you can also run them. In Dagster, running a dbt model corresponds to _materializing_ an asset. Materializing an asset means running some computation to update its contents in persistent storage. In this tutorial, that persistent storage is our local DuckDB database.
 
-defs = Definitions(assets=load_assets_from_modules([assets]), resources=resources)
-```
+To build your dbt project, i.e. materialize your assets, click the **Materialize all** button near the top right corner of the page. This which will launch a run to materialize the assets. When finished, the **Materialized** and **Latest Run** attributes in the asset will be populated:
```

Comment:
```suggestion
To build your dbt project, i.e. materialize your assets, click the **Materialize all** button near the top right corner of the page. This will launch a run to materialize the assets. When finished, the **Materialized** and **Latest Run** attributes in the asset will be populated:
```